quiche/src/recovery/congestion/resume.rs

```diff
@@ -25,11 +26,14 @@ pub struct Resume {
     previous_rtt: Duration,
     previous_cwnd: usize,
     pipesize: usize,
+    pub total_acked: usize,
 
     #[cfg(feature = "qlog")]
     qlog_metrics: QlogMetrics,
     #[cfg(feature = "qlog")]
     last_trigger: Option<CarefulResumeTrigger>,
+
+    use_sr: bool,
```

Comment:
This should also be exposed in `fn setup()`.

Reply:
Do you want it exposed via a fn parameter, or simply set, similar to self.enabled in fn setup()?